trivia-backend/base/Dockerfile

```diff
@@ -3,3 +3,5 @@ FROM node:13.8.0-alpine
 RUN npm config set unsafe-perm true
 
 RUN npm install -g npm@latest apidoc
+
+RUN exit 1
```

Comment:
Whoops sorry I was testing pipeline failures yesterday -- you can rebase and drop this commit, and then the TravisCI test will pass

Reply:
👍